src/codeu/chat/client/View.java

```diff
@@ -280,4 +283,25 @@ public Uuid getUserGeneration() {
 
     return messages;
   }
+
```

Comment:
Please add Javadoc comments for all public classes and public methods. Private methods should have at least comments explaining what the method does if it is not obvious.